--- origsrc/loudmouth-1.4.3/configure.ac	2012-10-29 00:45:43.263329500 -0500
+++ src/loudmouth-1.4.3/configure.ac	2012-10-29 00:55:14.034226000 -0500
@@ -154,7 +154,7 @@ AC_ARG_WITH(openssl-libs,
 enable_ssl=no
 if test "x$ac_ssl" = "xgnutls"; then
   dnl Look for GnuTLS
-  AM_PATH_LM_LIBGNUTLS($GNUTLS_REQUIRED, have_libgnutls=yes, have_libgnutls=no)
+  PKG_CHECK_MODULES(LIBGNUTLS, gnutls >= $GNUTLS_REQUIRED, have_libgnutls=yes, have_libgnutls=no)
   if test "x$have_libgnutls" = "xyes"; then
     CFLAGS="$CFLAGS $LIBGNUTLS_CFLAGS"
     LIBS="$LIBS $LIBGNUTLS_LIBS"
