--- origsrc/loudmouth-1.4.1/configure.ac	2008-08-01 01:34:09.406250000 -0500
+++ src/loudmouth-1.4.1/configure.ac	2008-08-01 01:32:34.390625000 -0500
@@ -15,12 +15,18 @@
 AC_CANONICAL_HOST
 
 os_win32=no
+platform_win32=no
 
 case "$host_os" in
   *mingw32*)
     os_win32=yes
 esac
 
+case "$host_os" in
+  *mingw32*|*cygwin*)
+    platform_win32=yes
+esac
+
 if test "$os_win32" = "yes"; then
   if test "$enable_static" = "yes" -o "$enable_static" = ""; then
     AC_MSG_WARN([Disabling static library build, must build as DLL on Windows.])
@@ -34,6 +40,8 @@
   enable_shared=yes
 
   CFLAGS="$CFLAGS -mms-bitfields"
+fi
+if test "$platform_win32" = "yes"; then
   LDFLAGS="$LDFLAGS -no-undefined"
 fi
 
